modules/osdk-cli-reference-generate-bundle.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-cli-reference.adoc
+
+[id="osdk-cli-reference-generate-bundle_{context}"]
+= bundle
+
+The `generate bundle` subcommand generates a set of bundle manifests, metadata, and a `bundle.Dockerfile` file for your Operator project. Typically, the `generate kustomize manifests` subcommand would be run first to generate the input link:https://kustomize.io/[Kustomize] bases that are consumed by the `generate bundle` subcommand.
```

Comment:
```suggestion
The `generate bundle` subcommand generates a set of bundle manifests, metadata, and a `bundle.Dockerfile` file for your Operator project. Typically, you run the `generate kustomize manifests` subcommand first to generate the input link:https://kustomize.io/[Kustomize] bases that are consumed by the `generate bundle` subcommand.
```

> ! consume verb
Do not use when a simpler term such as "use" is possible. "Consume" is acceptable when it is part of the
existing product terminology.

We'd been following the old guidance to use "consume" to mean "use up," so this instance might be fine. Please check.